src/main/java/com/gbtec/gameoflife/custom/GameOfLifeService.java

```diff
@@ -0,0 +1,47 @@
+package com.gbtec.gameoflife.custom;
+
+import com.gbtec.gameoflife.basic.GameOfLifeCommandProxy;
+
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GameOfLifeService extends GameOfLifeCommandProxy {
+
+    public GameOfLifeService(SimpMessagingTemplate simpMessagingTemplate) {
+        super(simpMessagingTemplate);
+    }
+
+    @Override
+    public void init() {
+        // Instead of "int[][]" you can also use "boolean[][]"
+        int[][] generationData = {
+                {0, 0, 0, 0, 0, 0, 0, 0},
+                {0, 0, 0, 0, 0, 0, 0, 0},
+                {0, 0, 0, 0, 0, 0, 0, 0},
+                {0, 0, 1, 0, 1, 0, 0, 0},
+                {1, 0, 0, 0, 0, 0, 1, 0},
+                {0, 1, 0, 0, 0, 1, 0, 0},
+                {0, 0, 1, 1, 1, 0, 0, 0},
+                {0, 0, 0, 0, 0, 0, 0, 0}
+        };
+
+        drawGeneration(generationData);
+
+        /* To generate random values use class "Random"
+         *
+         * Random random = new Random();
+         * random.nextBoolean(); // Generates "true" or "false"
+         * random.nextInt(0, 2); // Generates "0" or "1"
+         */
```

Comment:
Do not comment code out - rather provide an example like the following:
```suggestion
        // To generate random values use class "Random"
        Random random = new Random();
        random.nextBoolean(); // Generates "true" or "false"
        random.nextInt(0, 2); // Generates "0" or "1"
```

Reply:
But this example does nothing in this way ... i only tried to sow that java has a random class and you can get bool and int values from it and how to get it for the generation data correctly.

I can also just describe it in words instead of providing a snippet, but i don´t know if it is then more understandable. Moving it to README would probably be a better idea?

Reply:
Anyway, I would not use comments as examples - it promotes bad usage which we should not teach.
I would just omit that example in this case.